cri/v1alpha2/cri_types.go

```diff
@@ -12,8 +12,8 @@ type SandboxMeta struct {
 	// Config is CRI sandbox config.
 	Config *runtime.PodSandboxConfig
 
-	// Runtime is the runtime of sandbox
-	Runtime string
+	// RuntimeHandler is the runtime handler name of the pod.
+	RuntimeHandler string
```

Comment:
A question, the modification of this field's name will make this version of pouch incompatible with the old version of pouch. because it fails to recognize the `Runtime` field, fails to unmarshal from the meta json string stored in disk.
Just persisting the wording `Runtime`? @Starnop

Reply:
For backwards compatibility, I have updated it. Thanks for your advice.